_newsletter/issue-1/index.md

```diff
@@ -0,0 +1,40 @@
+---
+date: 2018-06-17
+issue: 1
+
+description: "The first ever issue of LearnK8s newsletter. Full of informative, educational and even funny stuff."
+
+excerpt: "The first ever issue of LearnK8s newsletter. Full of informative, educational and even funny stuff."
+
+open_graph:
+    description: "The first ever issue of LearnK8s newsletter. Full of informative, educational and even funny stuff."
+
+editor_pick:
+  - title: "The journey from monolith to Docker to Kubernetes: part 1"
+    url: "https://devops.college/the-journey-from-monolith-to-docker-to-kubernetes-part-1-f5dbd730f620"
+    image: ed_pick_1.png
+    summary: "'Why' and 'How' to go from a monolith app to dockerized one and eventually to run our app on a kubernetes cluster. In part one, the author discusses the benefits and the abilities of Docker and the app's code architecture."
+    tags: [docker, micro services]
+  - title: "Dissecting Kubernetes deployments"
+    url: "https://blog.heroku.com/dissecting-kubernetes-deployments"
+    summary: Twitter's team have been developing a design system they call Horizon. Ashlie Ford tracks its journey, from humble beginnings as a hack-day project.
+    tags: [deployments]
+
+projects:
+  - name: "kubernetes-helm/chartmuseum"
+    stars: 517
+    forks: 66
+    url: "https://github.com/kubernetes-helm/chartmuseum"
+    image: project_1.png
+    summary: "ChartMuseum is an open-source Helm Chart Repository written in Go (GoLang), with support for cloud storage backends, including Google Cloud Storage, Amazon S3, and Microsoft Azure Blob Storage."
+  - name: "pusher/k8s-auth-example"
+    stars: 20
+    forks: 6
+    url: "https://github.com/pusher/k8s-auth-example"
+    summary: "Example Kubernetes Authentication helper. This is an example of how to connect to an OIDC provider and authenticate users before configuring their kubeconfig."

```

Comment:
Not all newsletters will have all sections. And some newsletter will have more sections than others. The goal is to make sure we can add custom sections as content. I suggest that the file should be `.html` instead of `.md`. Any HTML after the front-matter will be part of the newsletter.
If you prefer, you could also structure it so that each section is a different include (like you've done for the author snippet). In that way a newsletter is a collection of `{% include tweet.html url="blabla" %}`.

Reply:
I was trying to make so easy to publish an issue that you won't have to write any HTML. I'll use includes in this manner:

To include an editor pick:

```html
{% include newsletter/editor_pick title="some title"
                                                      url="this.com"
                                                      summary="some summary"
                                                      image="image.png"
                                                      tags="comma,seperated,tags" %}
```

The section itself will need to be hardcoded in each issue. This includes adding favicons such as the "Watch out for these projects" section which has a GH icon on the heading.

New section parts (such as a GitLab project) will need to either ne hardcoded or a new include developed when it is needed.

Is this how you've envisioned it?

I'll update the plan on #GH-89

Reply:
Whatever is easiest to deliver the landing page before the 6th of July. I trust your decision if you suggest selecting frontmatter. We can always revisit later when the requirements are more clear.

Reply:
👍 Thanks, @danielepolencic I'll have it ready before the 6th. I'll proceed with my original plan.